packages/shared/lpc/tests/legacy_remap.test.ts

```
@@ -0,0 +1,102 @@
+          expect(derivedAssetId).toBe(legacyAssetId);
+        } else {
+          // No remap needed for this index — it's the same in both orders
+          const derivedAssetId = derivedSlot.variants[legacyIdx]?.assetId;
+          expect(derivedAssetId).toBe(legacyAssetId);
+        }
+      }
+    }
+  });
+
+  test('remap table has no duplicate target indices per slot', () => {
+    for (const [slot, remap] of Object.entries(LEGACY_INDEX_REMAP)) {
```

Biome (2.5.7)

[error] This variable slot is unused.

(lint/correctness/noUnusedVariables)